cms/models/permissionmodels.py

```diff
@@ -5,14 +5,16 @@
 from django.contrib.auth.models import Group, UserManager
 from django.contrib.sites.models import Site
 from django.core.exceptions import ImproperlyConfigured, ValidationError
-from django.utils.encoding import force_text, python_2_unicode_compatible
+from django.utils.encoding import force_text
 from django.utils.translation import ugettext_lazy as _
 
 from cms.models import Page
 from cms.models.managers import (PagePermissionManager,
                                  GlobalPagePermissionManager)
 from cms.utils.compat import DJANGO_1_11
 
+from six import python_2_unicode_compatible
```

Comment:
django 1.11 is the last version to keep python 2 support isn;t it?

Reply:
Regardless of which version it was, python 2 support shouldn't be maintained anymore 👍

Reply:
This is a backport of the original change for django 3.0. This has been done to have a direct comparison to the original change. There is another PR for django 3.1 and another for 3.2 which is a few PRs combined.